### PR TITLE
Hbb 41 bridge logic

### PR DESCRIPTION
**Pull request by fuvaji**
Bridge logic done

### CODE REVIEW

File: Assets/Scripts/BridgeManager.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BridgeManager : MonoBehaviour
+{
+
+    [SerializeField] private GameObject _bridgeParent;
+    [SerializeField] private Timer _timer;
+    // Start is called before the first frame update
```
Comment (cupoftea4) [inline Assets/Scripts/BridgeManager.cs:10]: Newline, comment

File: Assets/Scripts/BridgeManager.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BridgeManager : MonoBehaviour
+{
+
```
Comment (cupoftea4) [inline Assets/Scripts/BridgeManager.cs:7]: Empty line is obsolete

**Review by Oleh325 — APPROVED** [review]

File: Assets/Scripts/BridgeManager.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
```
Comment (cupoftea4) [inline Assets/Scripts/BridgeManager.cs:2]: Remove useless usings

File: Assets/Scripts/BridgeManager.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        _timer.OnDayNightTransition += SetBridgesActive;
+    }
+
+    private void OnDestroy()
+    {
+        _timer.OnDayNightTransition -= SetBridgesActive;
+    }
+
+    // Update is called once per frame
```
Comment (cupoftea4) [inline Assets/Scripts/BridgeManager.cs:21]: Remove comment

**Review by cupoftea4 — APPROVED** [review]

**Pull request merged**